video_xblock/video_xblock.py

```diff
@@ -43,6 +42,13 @@ class ContentStoreMixin(XBlock):
     from `xmodule.contentstore`
     """
 
+    @staticmethod
+    def import_from(module, klass):
```

Comment:
does it make sense to have it as a static method? It looks like an independant utility.